### PR TITLE
Update links.yml

### PR DESCRIPTION
**Pull request by CarrotManMatt**
As discussed in latest committee meeting there will be useful links added to this page about event signup forms etc.

### CODE REVIEW

**Review by CharlesESW — CHANGES_REQUESTED** [review]
Remove outdated Links:
Missing Semester
Great Uni Hack
Kainos open Evening

**Comment by CharlesESW** [comment]
Pls Bestie can you add Instagram, facebook, whatsapp community, ducks,

File: _data/links.yml [hunk 1]
```diff
@@ -5,3 +5,15 @@
 - text: Join our Discord!
   url: /discord
   image: /assets/images/discord-logo.png
+
+- text: "AFNOM's 'Missing Semester' signup form"
+  url: https://docs.google.com/forms/d/e/1FAIpQLSd0AqwJ774SfcQvd-j1bNQTeqIcBAjMEXFUro6hQbsmYYXIxw/viewform
```
Comment (Vishwas-Adiga) [inline _data/links.yml:10]: Could we replace this with `https://missingsemester.afnom.net/` please

**Review by MattyTheHacker — DISMISSED** [review]

**Review by MattyTheHacker — DISMISSED** [review]

**Review by MattyTheHacker — APPROVED** [review]

**Review by MattyTheHacker — DISMISSED** [review]

**Review by MattyTheHacker — APPROVED** [review]

File: _data/links.yml (hunk 1 above)
Comment (CarrotManMatt) [reply _data/links.yml]: Fixed!

**Review by MattyTheHacker — APPROVED** [review]

**Comment by CarrotManMatt** [comment]
This depends on #509 because the whatsapp link is referenced but does not yet exist.

**Review by MattyTheHacker — APPROVED** [review]

**Review by CharlesESW — APPROVED** [review]

**Pull request merged**